scripts/time_plot.py

```diff
@@ -1497,6 +1497,69 @@ def handle_websock_event(handlerkey,*args):
                 for thishandler in websockrequest_username.keys():
                     if (websockrequest_username[thishandler]==username):
                         send_websock_cmd('ApplyViewLayout('+'["'+'","'.join([fig['figtype'] for fig in html_viewsettings[username]])+'"]'+','+str(html_layoutsettings[username]['ncols'])+')',thishandler)
+        elif (args[0].startswith('holo')):#accepts 'holohh', 'holovv', 'holohv', 'holovh', 'holo 3' where 3 is m003 as explicit reference antenna, else first track antenna is chosen as reference antenna
```

Comment:
'holo 3' is presumably no longer valid here, since it gets rewritten to holohh?

Reply:
The comment is a bit misleading, from user side it should still work because holo is replaced with holohh in client.js

Reply:
client.js should separate the 3 as args[1]

Reply:
But what's received here will be 'holohh 3', right?

Reply:
I am expecting args=['holohh','3']

Reply:
```suggestion
        elif (args[0].startswith('holo')):#accepts 'holohh', 'holovv', 'holohv', 'holovh', 'holohh 3' where 3 is m003 as explicit reference antenna, else first track antenna is chosen as reference antenna
```